missions/2PzD_Template.VR/customization/loadouts/US/loadoutUS42.sqf

```diff
@@ -152,12 +153,13 @@
         [GEN_Gren_Smoke_W,1] call Olsen_FW_FNC_AddItem;
     }];
 
-    //Messenger
+    //Radio Operator
```

Comment:
I think you might have to double check the Composition on this one. The Platoon squad may have two of these Messengers, and we don't want two radiomen. Either way, it would be a cleaner solution to make a proper radioman case.

Reply:
I may just revert the changes to this and keep the messenger, based to the fact we never use Radio operators and messengers can be more usefull, We can go over and add a radio operator on a later date when i feel like messing with the compostion ^^,

Reply:
Sure